bindings/Python/py11File.tcc

```diff
@@ -18,13 +18,54 @@ namespace adios2
 namespace py11
 {
 
+static Dims py_strides(const Dims &shape, ssize_t itemsize, bool has_step_dim)
```

Comment:
let's discuss the need for `static`, also we use camelCase style `const bool hasStepDim`

Reply:
will do on the camel case. 

This one is static since it's not meant to be used outside of that source file. I could use an anonymous namespace instead to be more C++-y, I guess, if you prefer.

Reply:
@germasch yes, an anonymous namespace is fine, help us make the distinction between static functions as "only used here" and those static member functions that handle static members.